source/server/overload_manager_impl.cc

```diff
@@ -72,15 +114,21 @@ Stats::Gauge& makeGauge(Stats::Scope& scope, absl::string_view a, absl::string_v
 
 OverloadAction::OverloadAction(const envoy::config::overload::v3::OverloadAction& config,
                                Stats::Scope& stats_scope)
-    : active_gauge_(
-          makeGauge(stats_scope, config.name(), "active", Stats::Gauge::ImportMode::Accumulate)) {
+    : state_(OverloadActionState::inactive()),
+      active_gauge_(
+          makeGauge(stats_scope, config.name(), "active", Stats::Gauge::ImportMode::Accumulate)),
+      scale_value_gauge_(makeGauge(stats_scope, config.name(), "scale_value",
```

Comment:
The stat seems to be a percent.  Should the name be: scale_percent_gauge_ and "scale_percent" ?